### DISCUSSION TITLE
[idea] option to ignore cd covers/artwork

### DISCUSSION THREAD

**Discussion by y1111** [FossifyOrg/General-Discussion#570] [Ideas]
to tick ignore it for the app not even loading covers . it can save memory used .
and also when we swipe the notifications the artwork appears there . i get  some people may like but i use dark black themes for saving battery (amoled) so it would be goo that the notification entry to move between track to be just themed like the app , not showing any covers

**Comment by ghost** [comment]
I need this too. I dowloaded all of my stuff ( ~330 songs ) from PipePipe and none of them have covers. They all have them annoying ahh headphone icon

**Comment by Aga-C** [comment]
Please create a [feature request](https://github.com/FossifyOrg/Music-Player/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml) for this.
